Library/Homebrew/attestation.rb

```diff
@@ -104,6 +107,13 @@ def self.check_attestation(bottle, signing_repo, signing_workflow = nil, subject
         # Even if we have credentials, they may be invalid or malformed.
         raise GhAuthNeeded, "invalid credentials" if e.status.exitstatus == 4
 
+        gh_version = Version.new(system_command!(gh_executable, args: ["--version"], print_stderr: false)
+                                 .stdout.match(/\d+(?:\.\d+)+/i).to_s)
+        if gh_version < GH_ATTESTATION_MIN_VERSION
+          raise e,
+                "`gh` is too old, you must run `brew upgrade gh` to continue."
```

Comment:
```suggestion
                "`#{gh_executable}` is too old, you must run upgrade it to continue."
```

maybe we should automatically upgrade it if it's an installed formula?